browser/ui/brave_browser_command_controller.cc

```diff
@@ -249,15 +249,16 @@ void BraveBrowserCommandController::UpdateCommandForBraveVPN() {
     UpdateCommandEnabled(IDC_SHOW_BRAVE_VPN_PANEL, false);
     UpdateCommandEnabled(IDC_BRAVE_VPN_MENU, false);
     UpdateCommandEnabled(IDC_TOGGLE_BRAVE_VPN_TOOLBAR_BUTTON, false);
+    UpdateCommandEnabled(IDC_TOGGLE_BRAVE_VPN_TRAY_ICON, false);
     UpdateCommandEnabled(IDC_SEND_BRAVE_VPN_FEEDBACK, false);
     UpdateCommandEnabled(IDC_ABOUT_BRAVE_VPN, false);
     UpdateCommandEnabled(IDC_MANAGE_BRAVE_VPN_PLAN, false);
     UpdateCommandEnabled(IDC_TOGGLE_BRAVE_VPN, false);
     return;
   }
-
   UpdateCommandEnabled(IDC_SHOW_BRAVE_VPN_PANEL, true);
   UpdateCommandEnabled(IDC_TOGGLE_BRAVE_VPN_TOOLBAR_BUTTON, true);
+  UpdateCommandEnabled(IDC_TOGGLE_BRAVE_VPN_TRAY_ICON, true);
```

Comment:
This command should be enabled only when wireguard feature is enabled?

Reply:
done